symfony/src/Controller/AuthController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Controller;
+
+use App\Form\MockLoginType;
+use App\Service\AuthService;
+use App\Service\EnvironmentService;
+use App\Service\UserService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\User\UserProviderInterface;
+
+/**
+ * @Route("/s")
+ */
+class AuthController extends AbstractController
+{
+    /**
+     * @Route("/login", name="login")
+     */
+    public function login(UserService $userService, Request $request, UserProviderInterface $userProvider, EnvironmentService $env, AuthService $authService, SessionInterface $session)
+    {
+        if ($this->getUser()) {
+            return $this->redirectToRoute('symfony_home');
+        }
+
+        if ($env->isLocal() && $userService->canMockLogin()) {
+            $loginForm = $this->createForm(MockLoginType::class);
+
+            $loginForm->handleRequest($request);
+
+            if ($loginForm->isSubmitted() && $loginForm->isValid()) {
+                // Set mock user and token for local development
+                $userService->setMockUser($loginForm->get('userName')->getData());
+                $user = $userProvider->loadUserByUsername($loginForm->get('userName')->getData());
+                $authService->setMockAuthToken($user);
+                $session->set('isLoginReturn', true);
+                return $this->redirect('/s');
+            }
+
+            return $this->render('mock-login.html.twig', [
+                'loginForm' => $loginForm->createView()
+            ]);
+        }
+
+        return $this->render('login.html.twig');
```

Comment:
For now, since we don't have any direction on what to put on the login page, should we just go ahead and redirect to the login start page?